functionsAndScope.js

```diff
@@ -0,0 +1,285 @@
+// # Function Exercises
+
+// ## Question 1
+
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+
+// Example:
+
+// Input: `12, 4`
+
+// Output: `12`
+
+function whosBigger (numeroUno, numeroDos) {
+    if (numeroUno > numeroDos) {
+        return numeroUno;
+    } else {
+        return numeroDos;
+    }
+}
+
+console.log("1) ", whosBigger(100,1000));
+console.log("- - - - - - - - - - - - - - - - - - - - - - - - - - -");
+
+// ## Question 2
+
+// Write a function named getDogAge that takes one argument: a dog's age in human years. The function returns the dog's age in dog years. The conversion rate is 1 human year to 7 dog years.
+
+// Example:
+
+// Input: `4`
+
+// Output: `28`
+
+function getDogAge (doggoAge) {
+    return doggoAge / 7;
+}
+
+console.log("2) ", getDogAge(28));
+console.log("- - - - - - - - - - - - - - - - - - - - - - - - - - -");
+
+// ## Question 3
+
+// Write a function `arraySum` that takes in an array and returns the sum of all the elements.
+
+// Example:
+
+// Input: `[13,2,4,-2]`
+
+// Output: `17`
+
+function arraySum (arr) {
+    let sum = 0;
+    arr.forEach(element => {
+        sum += element;
+    });
+    return sum;
+}
+
+console.log("3) ", arraySum([1,2,3,4,5]));
+console.log("- - - - - - - - - - - - - - - - - - - - - - - - - - -");
+
+// ## Question 4
+
+// Write a function that takes in any two numbers and return their sum.
+
+// Example:
+
+// Input: `14, 5`
+
+// Output: `19`
+
+const addMeUp = (one, two) => one + two;
+
+console.log("4) ", addMeUp(23, 24));
+console.log("- - - - - - - - - - - - - - - - - - - - - - - - - - -");
+// ## Question 5
+
+// Write a function takes in any number grade and returns a corresponding letter grade.
+
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+
+
+// Example:
+
+// Input: `88`
+
+// Output: `"B"`
+
+const passOrFail = (grade) => {
+    return grade === 100 ? "A"
+        : grade > 89 ? "A"
+        : grade > 79 ? "B"
+        : grade > 69 ? "C"
+        : grade > 64 ? "D"
+        : "F";
+}
+
```

Comment:
I love the use of the ternary operations ! Great job! 

When the grade = 100 -- The return should be "A+"